src/components/policy/policy_regular/src/policy_table/validation.cc

```diff
@@ -136,6 +136,24 @@ bool ModuleConfig::Validate() const {
       return false;
     }
   }
+
+  if (endpoints.empty()) {
```

Comment:
@LuxoftAKutsan @AKalinich-Luxoft 
It is not mandatory if endpoints.valid() is called.